code/modules/mob/living/carbon/monkey/combat.dm

```diff
@@ -0,0 +1,433 @@
+
+
+/mob/living/carbon/monkey
+	var/aggressive=0
```

Comment:
Still not used. Mostly want this implemented so admin created monkey hordes are dangerous. But removal also works.